pnpm-workspace.yaml

```diff
@@ -1,2 +1,7 @@
 packages:
   - 'packages/*'
+  - 'app-directory/*'
+  - 'edge-api-routes/*'
+  - 'edge-functions/*'
+  - 'plop-templates/*'
+  - 'solutions/*'
```

Comment:
Those changes probably aren't what we want. All of the examples are not mono repo packages, but examples on their own including **mono repo examples**. Deps shouldn't be installed globally for every example. `/packages/*` folder contains the actual packages we want to control. 

When people want to try an example, they will only install the dependencies for that example. In addition, we have Python examples that will not be served by `pnpm`

Reply:
> they will only install the dependencies for that example

can you explain how to do this? running `pnpm i` in any folder tries to install them at the root

Reply:
I was running `npm i` inside the examples as it is in the readme but yes, it looks like it's broken for `pnpm` when run inside the examples.